Tests/DiscoveryV2IntegrationTests.cs

```diff
@@ -50,6 +50,8 @@ public class DiscoveryV2IntegrationTests
         public void OneTimeSetup()
         {
             LogSystem.InstallDefaultReactors();
+            analyzeDocumentFile = Application.dataPath + "/Watson/Tests/TestData/DiscoveryV1/exampleConfigurationData.json";
```

Comment:
Are these running? I think we can exclude disco v2 tests as we don't have a persistent instance